docs/api/ref/DesktopAgent.md

```diff
@@ -91,6 +94,30 @@ const contactListener = await fdc3.addContextListener('fdc3.contact', (contact,
 - [`Context`](Types#context)
 - [`ContextHandler`](Types#contexthandler)
 
+### `addEventListener`
+````ts
+addEventListener(type: FDC3EventType  | null, handler: EventHandler): Promise<Listener>;
+````
```

Comment:
```suggestion
```ts
addEventListener(type: FDC3EventType  | null, handler: EventHandler): Promise<Listener>;
```
```
Currently has quadruple backticks

Reply:
@hughtroeger FYI this suggestion was dropping the last line due a clash between the suggestion syntax and the triple backticks in what was being corrected. To avoid that use quadruple backticks in the suggestion!